shared/desktop/renderer/style.css

```diff
@@ -451,3 +451,11 @@ body {
 .darkMode .headerTitle:hover .addInviteAndLinkBox {
   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.8);
 }
+
+.checkboxRow:hover .checkCircle {
```

Comment:
parallel implementation of this in master, see if it works here.

Reply:
Oh nice, didn't see this since it was in a difference CSS place